README.md

```diff
@@ -56,6 +56,7 @@ override['ssh-hardening']['ssh']['server']['listen_to'] = node['ipaddress']
 * `['ssh-hardening']['ssh']['server']['allow_tcp_forwarding']` - `false`. Set to `true` to allow TCP Forwarding
 * `['ssh-hardening']['ssh']['server']['allow_agent_forwarding']` - `false`. Set to `true` to allow Agent Forwarding
 * `['ssh-hardening']['ssh']['server']['allow_x11_forwarding']` - `false`. Set to `true` to allow X11 Forwarding
+* `['ssh-hardening']['ssh']['server']['permit_tunnel']` - `false` to disable tun device forwarding. Set to `true` to allow tun device forwarding. Other accepted values: 'yes', 'no', 'point-to-point', 'ethernet', see man sshd\_config pages for exact behaviors. Do note you'll also need to allow TCP forwarding.
```

Comment:
can you please change it this way in order to match the style of README.md:

```markdown
* `['ssh-hardening']['ssh']['server']['permit_tunnel']` - `false`. Set to `true` to allow tun device forwarding. Other accepted values: 'yes', 'no', 'point-to-point', 'ethernet', see `man sshd_config` for more details. Note: `allow_tcp_forwarding` should be enabled together with this option.
```

Something like this?